docs/code-of-conduct.rst

```diff
@@ -6,16 +6,30 @@ Code of Conduct
    :depth: 1
    :backlinks: none
 
-Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.
+Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation. Everybody who participates in our community in one way or another is required to conform to this Code of Conduct (CoC). This includes online communications, meetup and conference attendees, speakers, sponsors, founders, moderators, organisers and volunteers.
 
-Diversity is one of our huge strengths, but it can also lead to communication issues. To support a welcoming environment for all, regardless of individual differences, we have a few ground rules that we ask people to adhere to when they participate in this community activities. These rules apply equally to founders, organizers, moderators, sponsors, and affiliates -- in short, to all participants.
+Why do we have a Code of Conduct?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This isn't an exhaustive list of things that you must do, or can't do. Rather, take it in the spirit in which it's intended. It's a guide to make it easier to enrich all of us and the technical communities in which we participate, and which we represent.
+Our goals with having this Code of Conduct are:
```

Comment:
I like this section. We always get a lot of "oh this place must be scary if you're so worried about conduct" responses, and this is a good framing against that.